tests/expectations/core/test_expect_column_values_to_be_in_set.py

```diff
@@ -198,4 +198,4 @@ def test_expect_column_values_to_be_in_set_no_set(
     with pytest.raises(
         great_expectations.exceptions.exceptions.InvalidExpectationConfigurationError
     ):
-        result = validator.expect_column_values_to_be_in_set(column="a")
```

Comment:
Do we not want to save the result returned from the expectation call?

Reply:
ruff is removing this because it wasn't being used.
It could be useful to keep just to show that this method is actually returning something.

Reply:
Yea, and potentially useful when inspecting with debugger. Similar for one `except Exception as x` that just does `pass`. I suppose someone can just add those in if needed during some debugging.

Reply:
Added the `result` variable back.
I'm leaving the exceptions as `ruff` changed them, although I'm not opposed to those debugging logs.
If they are useful I think we should commit them.